src/pages/[platform]/build-a-backend/storage/download/index.mdx

```diff
@@ -129,12 +129,35 @@ await getUrl({ key: 'filename.txt', options: { expiresIn: 60 } });
 
 Download a file to in-memory buffer.
 
+<BlockSwitcher>
+<Block name="Path">
 ```javascript
 import { downloadData } from 'aws-amplify/storage';
 
 // Downloads file content to memory
 const { body, eTag } = await downloadData({
-  key,
+  path: filename,
```

Comment:
Since we are renovating the examples... This `filename` variable is not seen anywhere else in the example, can we use a actually string value here instead? (I think `/album/2024/1.jpg` will be more intuitive in the context of using the path parameter)

Reply:
Yeah i had a string too and then made it filename for consistency. Yes i think we can make it all actual string